src/scripts/services/password_manager.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * @param length a number involving the length of the password
+ * @returns a string of set length containing random charachters
+ */
+export const generatePassword = (length: number): string => {
+  return length > 0
+    ? randomChar() + generatePassword(length - 1)
+    : randomChar();
+};
+
+// eslint-disable-next-line functional/functional-parameters
+export const randomChar = (): string => {
+  const ordinal = Math.round(Math.random() * 61);
```

Comment:
Let's bring in a third-party library to do this for us, replacing this entire file. Even though these are test users, we should be in the habit of generating really random password *and* we can anticipate that we will have to generate passwords with special characters.

Reply:
A separate third-party password generator was implemented